### PR TITLE
Add email invite feature

### PR DESCRIPTION
**Pull request by sladyn98**
### Description

This feature attempts to send an email to the user after generating the token

## Checklist

- [ ] Everything passes when running `mix test`
- [ ] Ran `mix format`
- [ ] No frontend compilation warnings

### CODE REVIEW

File: assets/src/components/account/AccountOverview.tsx [hunk 1]
```diff
@@ -82,6 +82,17 @@ class AccountOverview extends React.Component<Props, State> {
     }
   };
 
+
+  sendEmailToUser = async (email:string) => {
+    try {
+      const {id: token} = await API.generateUserInvitation();
+      // Call api from elixir code
```
Comment (sladyn98) [inline assets/src/components/account/AccountOverview.tsx:89]: @reichert621 Need help on how to call the elixir function. Should I add it in the api.ts and then call it but could not understand the flow there

Comment (cheeseblubber) [reply assets/src/components/account/AccountOverview.tsx:89]: I would recommend going through https://elixir-lang.org/getting-started/introduction.html the documentation is pretty good

**Comment by reichert621** [comment]
@sladyn98 can you please mark this PR as a draft if it's not ready for review yet?

if this is your first time working with Phoenix, it might be useful to check out these docs first: https://hexdocs.pm/phoenix/overview.html

File: assets/src/components/account/AccountOverview.tsx (hunk 1 above)
Comment (sladyn98) [reply assets/src/components/account/AccountOverview.tsx:89]: Thanks I did a quick run through.I just need some understanding on should this be just a normal function call from email.ex by importing it. ?

**Comment by sladyn98** [comment]
@reichert621 Thanks I went through a youtube video as well. Just need some additional help on  this. Once I get a feel for it I should be able to contribute better.

**Comment by reichert621** [comment]
@sladyn98 this might be a good PR for reference: https://github.com/papercups-io/papercups/pull/220

**Comment by reichert621** [comment]
closing for now, feel free to open up a PR once code is ready for review!

**Pull request closed without merging**